backend/src/test/nl/tudelft/ti2806/riverrush/domain/entity/AnimalTest.java

```diff
@@ -3,6 +3,7 @@
 import nl.tudelft.ti2806.riverrush.domain.entity.state.AnimalInAir;
```

Comment:
Please fix this line: https://github.com/StreetLampStudios/RiverRush/blob/feature/moretesting/backend/src/test/nl/tudelft/ti2806/riverrush/domain/entity/AnimalTest.java#L16